quickevent/app/quickevent/plugins/Runs/src/runflagsdialog.h

```diff
@@ -26,6 +26,8 @@ class RunFlagsDialog : public QDialog
 	Ui::RunFlagsDialog *ui;
 	RunsTableModel *m_model;
 	int m_row;
+	bool m_isDisqualified;
+	void updateStatusText();
```

Comment:
please don't mix fields and methods declarations

Reply:
I was always wondering, why there are multiple `private` and `public` blocks in header files. now i do understand :smile:

Reply:
This is not explicitly mentioned in code style, but we are using Qt code style, if explicit rule is not stated.